src/mmapped_plv.hpp

```diff
@@ -0,0 +1,54 @@
+// Copyright 2019-2020 libsbn project contributors.
+// libsbn is free software under the GPLv3; see LICENSE file for details.
+//
+// RAII class for partial likelihood vectors that are mmapped to disk.
+//
+// This class is to allocate a very large partial likelihood vector in virtual memory
+// and then cut it up (via Subdivide) into a vector of partial likelihood vectors.
+
+#ifndef SRC_MMAPPED_PLV_HPP_
+#define SRC_MMAPPED_PLV_HPP_
+
+#include "eigen_sugar.hpp"
+#include "mmapped_matrix.hpp"
+
+using NucleotidePLV = Eigen::Matrix<double, 4, Eigen::Dynamic, Eigen::ColMajor>;
```

Comment:
Should we create a constant for number of bases, 4? It seems to be used multiple places within this file and few places outside of this file as well.